octobot_commons/config_manager.py

```diff
@@ -289,51 +273,33 @@ def clear_dictionaries_by_keys(dict_dest, dict_src):
     return dict_dest
 
 
-def _update_activation_config(to_update_data, current_config, config_file_path, config_file, deactivate_others):
-    from octobot_commons.tentacles_management.class_inspector import get_class_from_string, evaluator_parent_inspection
+def update_activation_config(to_update_data, current_config, deactivate_others):
```

Comment:
I don't understand why this is not handled by the Tentacle Manager ?

Reply:
Right, should be there, i'll change it